src/sdl2/video.rs

```diff
@@ -1533,6 +1553,25 @@ impl Window {
         Ok(())
     }
 
+    #[doc(alias = "SDL_SetWindowShape")]
+    pub fn set_window_shape<S: AsRef<SurfaceRef>>(&mut self, shape: S) -> bool {
```

Comment:
I've looked at the API a bit, either you generalize this API with `WindowShapeMode`, `binarizationCutoff`, etc, or you name this function something more specific like `set_window_shape_alpha`, with as parameter only `binarizationCutoff`.

A doc comment saying that this can only be used if `shaped` is true on the builder would also be nice (and if you go for the latter, an explanation of `binarizationCutoff` for sane values, like 0xFF)

Reply:
Fixed this as well (and adding binaryationCutOff as parameter)